qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VisitSteps.java

```diff
@@ -121,5 +134,71 @@ public void clickOnTheEndVisitButton() {
     @Then("the system ends the visit")
     public void systemEndsVisit() {
     	assertTrue(textExists("No active visit"));
-    }   
+    }
+
+    @When("user clicks on the Active Visits app from the home page")
+    public void loadActiveVisitsPage() {
+    	activeVisitsPage = homePage.goToActiveVisitsSearch();
+    }
+
+    @And("user searches for the patient with active visit using patient Identifier")
+    public void searchForPatientUsingPatientIdentifier() {
+    	activeVisitsPage.search(patient.identifier);
+    	activeVisitsPage.waitForPageToLoad();
```

Comment:
Do we really need to call `waitForPageToLoad()` here?

Reply:
> Do we really need to call `waitForPageToLoad()` here?

@kdaud I thought it would be necessary but I can remove it.